gridcapa-swe-adapter-app/src/main/java/com/farao_community/farao/swe/adapter/app/SweAdapterListener.java

```diff
@@ -97,7 +100,7 @@ private SweFileResource getFileRessourceFromInputs(List<ProcessFileDto> listInpu
                 .filter(p -> p.getFileType().equals(type))
                 .findFirst()
                 .orElseThrow(() -> new SweAdapterException("No file found for type " + type));
-        return new SweFileResource(input.getFilename(), input.getFileUrl());
+        return new SweFileResource(input.getFilename(), minioAadpter.generatePreSignedUrlFromFullMinioPath(input.getFilePath(), 1));
```

Comment:
In case of error when generatingPresignedUrl (could generate a Runtime exception), we don't have any information in the HMI to know why the task was not "running". I think it's better to make the error appears. I got the error in local when testing before updating the postgres DB

Reply:
I am not sure how to throw this exception in the front